SessionMessagingKit/Database/Models/Attachment.swift

```diff
@@ -788,13 +789,30 @@ extension Attachment {
     public var isText: Bool { MIMETypeUtil.isText(contentType) }
     public var isMicrosoftDoc: Bool { MIMETypeUtil.isMicrosoftDoc(contentType) }
     
+    public var documentFileName: String {
```

Comment:
Do we want to localise these default file names?

Reply:
Yea probably but they already weren't localised so I wanted to try to minimise changes - since the `Strings` project is progressing separately in the background I wanted to avoid adding to much additional localisation since it's going to be re-done as it's own project in the future

Hopefully with the updated `LintLocalizableStrings` it'll be a lot easier to identify all of the unlocalised strings that are missing and we can look to add them as part of the `Strings` project